openqabot/loader/repohash.py

```diff
@@ -25,8 +25,7 @@ def get_max_revision(
 
     for repo in repos:
 
-        # don't use openSUSE-SLE
-        if repo[0] == "openSUSE-SLE":
+        if repo[0] in ("openSUSE-SLE", "openSUSE-Leap-Micro"):
```

Comment:
well, was not clear in before, still not clear why these should be handled differently, better to use a function with good name describing that

Reply:
because SLE incidents repo's have different structure than openSUSE.
And we handle only these two ... Right it can be simpler

Reply:
much better, thank you